### DISCUSSION TITLE
HELP QUESTION What's the privacy/security skinny (e.g. bots)?

### DISCUSSION THREAD

**Discussion by ldexterldesign** [slskd/slskd#508] [Q&A] [answered]
👋 @jpdillingham,

I was going to tell [this] Karen to (not so politely) jog on but I thought I would give the benefit of the doubt and check with an expert - before I consider replying do you have any input?

I run/host the app in a local Docker container

Aside, may I add: you run this project like a well oiled machine 🙌

Hope to hear back

Sincerely

[this]: https://mastodon.social/@teledyn/108449934192532561

**Comment by jpdillingham** [comment, marked as answer]
I appreciate the kind words :)

Personally I go out of my way to avoid exposing usernames and even actual track in screenshots and copy/pastes (but am not always perfect) to avoid drawing any unwanted attention to other users or Soulseek itself.
